.travis.yml

```diff
@@ -2,10 +2,8 @@ language: python
 dist: xenial
 sudo: required
 python:
-  - "3.5"
-  - "3.6"
-  - "3.7"
   - "3.8"
+  - "3.9"
```

Comment:
We are only using `3.8` and `3.9` in our builds, so there's no reason to include the older versions for Travis.